atom/browser/web_contents_preferences.h

```diff
@@ -62,6 +67,9 @@ class WebContentsPreferences
 
   // Set preference value to given bool if user did not provide value
   bool SetDefaultBoolIfUndefined(const base::StringPiece& key, bool val);
+  bool SetDefaultBoolIfUndefined(const base::StringPiece& key,
```

Comment:
you don't need to add a new method, just use a default value `DeprecationStatus status = Stable`